### ISSUE TITLE
"ExprVectorYawPitch" issues

### ISSUE THREAD

**Issue by jaylawl** [SkriptLang/Skript#877]
I'm trying this:
`add a random number between -1 and 1 to vector yaw of victim's vector`

https://snag.gy/RoWE28.jpg

Syntax learned from http://bensku.github.io/Skript/docs.json. I've had similar issues when messing the new vectors before. Always things that should work to common sense, but produce a similar parser message.

This works however:
`broadcast "%vector yaw of victim's vector%"`
Curious enough, the returned value is always a yaw between 340 and 345, no matter which direction the entity is going.

**Comment by bensku** [comment]
Oh wonderful, vector bugs again...

**Comment by bensku** [comment]
Adding scalar to vector is not supported currently.

**Comment by jaylawl** [comment]
This is straight from the docs.json. So these examples were added prematurely? Just trying to understand here. In that case, what else would i add?
```
add 45 to vector yaw of {_v}
subtract 45 from vector pitch of {_v}
```

**Comment by TheBentoBox** [comment]
It may be because you're doing it all in one line. Try splitting it so you get the victim's location vector on a different line and see if it helps:

```
set {_v} to victim's vector
add (random number between -1 and 1) to vector yaw of {_v}
```

**Comment by bensku** [comment]
Hm, the examples seem to be just plain wrong here. But I would rather implemnt this feature (or fix it) instead or just editing docs.

**Comment by TPGamesNL** [comment]
Fixed in #4064
